RctGcs/ui/display.py

```diff
@@ -1,47 +1,50 @@
-import configparser
 import json
 import logging
 import queue as q
 from functools import partial
 
 import utm
-from PyQt5.QtWidgets import (QFileDialog, QGridLayout, QLabel, QMainWindow,
-                             QPushButton, QScrollArea, QVBoxLayout, QWidget)
-from RCTComms.transport import RCTTCPServer
+from PyQt5 import QtCore, QtWidgets
+from RCTComms.comms import gcsComms
+from RCTComms.transport import RCTAbstractTransport, RCTTransportFactory
 
-import RctGcs.rctCore
-from RctGcs.config import get_config_path, get_instance
-from RctGcs.ui.controls import *
-from RctGcs.ui.map import *
+from RctGcs.config import ConnectionMode, get_config_path, get_instance
+from RctGcs.rctCore import (EXTS_STATES, OUTPUT_DIR_STATES, RCT_STATES,
+                            SDR_INIT_STATES, Events, MAVModel, NoActiveModel)
+from RctGcs.ui.controls import CollapseFrame, SystemSettingsControl
+from RctGcs.ui.map import MapOptions, StaticMap, WebMap
+from RctGcs.ui.popups import ConnectionDialog, UserPopups
```

Comment:
ConfigDialog missing (gets called on line 532 when user clicks "Edit Configuration Settings", last dropdown item in UI)

Reply:
fixed in f6d403757f76ff3a41d209b67bffa41204e4be63